### DISCUSSION TITLE
Wazuh on ARM64

### DISCUSSION THREAD

**Discussion by l0rddarkf0rce** [wazuh/wazuh-docker#927] [Q&A] [answered]
Are there any plans on making Wazuh-docker compatible with ARM64 platforms?

**Comment by rauldpm** [comment, marked as answer]
Hello, before we can support wazuh-docker on ARM64, we have to solve a compatibility problem with Filebeat 7.10.2 and the ARM architecture, since this version does not have a package for ARM, you can check the status of said development at the following issue: https://github.com/wazuh/wazuh-packages/issues/1637

Once this is resolved, the repository would be ready to be adapted to said architecture, since the Wazuh manager image incorporates both Wazuh manager and Filebeat
